src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -104,6 +109,29 @@ private void setDataFormat(IList<DataFormatParameter> formatParameters)
          View.SetMappingSource(_mappings);
          ValidateMapping();
          InitializeErrorUnit();
+         setDimensionsForMappings();
+      }
+
+      private void setDimensionsForMappings()
+      {
+         foreach (var mapping in _mappings)
+         {
+            var mappingColumn = (mapping.Source as MappingDataFormatParameter)?.MappedColumn;
+            if (mappingColumn?.Unit == null) 
+               continue;
+
+            //initial settings for fraction dimension
+            if (mapping.ColumnInfo.DefaultDimension?.Name == Constants.Dimension.FRACTION && 
+                mappingColumn.Unit.ColumnName.IsNullOrEmpty() && 
+                mappingColumn.Unit?.SelectedUnit == UnitDescription.InvalidUnit)
+            {
+               mappingColumn.Dimension = mapping.ColumnInfo.DefaultDimension;
+               mappingColumn.Unit = new UnitDescription("");
+               continue;
+            }
+
+            mappingColumn.Dimension = !mappingColumn.Unit.ColumnName.IsNullOrEmpty() ? null : _dimensionFactory.DimensionForUnit(mappingColumn.Unit.SelectedUnit);
```

Comment:
Well, on line 120 there is a 
`if (mappingColumn?.Unit == null) 
               continue;`
meaning at this point in the code, mappingColumn cannot be null anymore, right? Here the check is on ColumnName  and it could be empty string (not null)

Reply:
oh yeah missed it in the diff. Thanks